debian-setup.sh

```diff
@@ -25,10 +25,18 @@ apt-get install -y -qq \
     libssl-dev \
     make \
     openssl \
-    qemu-system-arm \
-    qemu-system-x86 \
     >/dev/null
 
+
```

Comment:
I think this extra line should go (or two spaces should be added in other places).